Src/WitsmlExplorer.Frontend/components/Modals/LogPropertiesModal.tsx

```diff
@@ -55,41 +55,32 @@ const LogPropertiesModal = (props: LogPropertiesModalInterface): React.ReactElem
                 id="uid"
                 label="uid"
                 value={editableLogObject.uid}
-                fullWidth
                 disabled={editMode}
                 required
-                error={!validText(editableLogObject.uid)}
```

Comment:
The equivalent of `error={!validText(editableLogObject.uid)}` in EDS is `variant={validText(editableLogObject.uid) ? undefined : "error"}`

Reply:
Fixed